multiqc/modules/custom_content/custom_content.py

```diff
@@ -202,10 +202,12 @@ def dict_constructor(loader, node):
                         hs[k] = v
                 gsheaders = hs
 
-            # Add namespace if not specified
-            for k in gsheaders:
-                if 'namespace' not in gsheaders[k]:
-                    gsheaders[k]['namespace'] = c_id
+            # Add namespace and description if not specified
+            for h in gsheaders:
+                if 'namespace' not in gsheaders[h]:
+                    gsheaders[h]['namespace'] = mod['config'].get('namespace', k)
+                if 'description' not in gsheaders[k]:
+                    gsheaders[h]['description'] = mod['config'].get('description', "General stat for custom module {}".format(k))
```

Comment:
I think it's better to leave this blank. The hover text always begins the with the `namespace`, which we have already collected above. So if undefined, it should already default to something pretty sensible. I don't think it's useful for people reading the report to know that the data was parsed with custom content.

Correct me if I'm wrong and the default hover text looks bad without this.

Reply:
No, I agree too, I was not so certain myself if this would be completely helpful. Just thought could get your opinion 😉

The default _description_ would be the same as column name. Which I guess is suffice 🤷‍♂️ , so I will remove this